.github/ISSUE_TEMPLATE/bug.yaml

```diff
@@ -0,0 +1,119 @@
+name: 🐞 bug fix
+description: "file a bug in noop services"
+title: "[Bug]: "
+labels: ["bug", "triage"]
+assignees:
+  - noop-cloud/noop
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ## Thanks for taking the time to fill out this bug report!
+  - type: markdown
+    attributes:
+      value: |
+        - Please review open issues labeled as `bugs` to ensure your issue is not a duplicate.
+        - For general questions, ideas and concerns that are not related to a bug, visit our [discussions page](https://github.com/noop-cloud/supreme-garbanzo/discussions)
+  - type: input
+    id: contact
+    attributes:
+      label: Contact Details
+      description: How can we get in touch with you if we need more info?
+      placeholder: ex. email@example.com
+    validations:
+      required: false
+  - type: dropdown
+    id: affected-os
+    attributes:
+      label: Operating System
+      description: What Operating System(s) have you experienced this bug on?
+      multiple: true
+      options:
+        - Windows
+        - MacOS
+    validations:
+      required: false
+  - type: dropdown
+    id: affected-arch
+    attributes:
+      label: Architecture
+      description: What machine architecture(s) have you experienced this bug on?
+      multiple: true
+      options:
+        - amd64
+        - arm64
+    validations:
+      required: false
+  - type: dropdown
+    id: affected-browsers
+    attributes:
+      label: Affected Browsers
+      description: If using the web application, which browsers have you experiened this bug in?
+      multiple: true
+      options:
+        - Firefox
+        - Chrome
+        - Safari
+        - Microsoft Edge
+    validations:
+      required: false
+  - type: dropdown
+    id: application-variant
+    attributes:
+      label:  Noop Variant
+      description: Which variant(s) of Noop are you experiencing this bug?
+      multiple: true
+      options:
+        - Web
+        - Desktop
+  - type: dropdown
+    id: application-mode
+    attributes:
+      label:  Noop Mode
+      description: Which run time mode(s) of Noop are you experiencing this bug?
+      multiple: true
+      options:
+        - Local Development
+        - Cloud Platform
+  - type: dropdown
+    id: affected-versions
+    attributes:
+      label:  Noop Version
+      description: What version(s) of Noop have you experienced this bug with?
+      multiple: true
+      options:
+        - 0.9.33 (default)
```

Comment:
Is it possible for people to add entries if they don't exist? I'd hate to have to manage a version list here 🙈

Reply:
> Is it possible for people to add entries if they don't exist? I'd hate to have to manage a version list here 🙈

Good question, will check if that feature exists in the YAML DSL they have for issues. It's still a "beta" feature in GH

Reply:
It appears that option is currently unsupported

[current features](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms)